src/containers/Password/ForgotPassword.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react';
+import {
+  Modal, Row, Button, Input
+} from 'react-materialize';
+import '../Login/Login.scss';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { sendForgotPassword } from '../../requests/PasswordRequests';
+
+/**
+ *
+ *
+ * @class Login
+ * @extends {Component}
+ */
+class ForgotPassword extends Component {
+  /**
+   * @member of Login
+   */
+
+  /**
+   * @description - This method hides the login modal
+   * @returns {object} null
+   * @memberof Login
+   */
+
+  /**
+   * @description - This method runs when the component upodates
+   * @returns {string} response
+   * @memberof ForgotPassword
+   */
+  componentDidUpdate() {
+    if (this.props.loading) {
+      if (this.props.errorMessage) {
```

Comment:
this is the error message that shows if the forgot password email was sent successfully or not